packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -200,3 +264,49 @@ function issueToString(issue: OperationOutcomeIssue): string {
     JSON.stringify(issue)
   );
 }
+
+async function convertFHIRBundleToMedicalRecord({
+  bundle,
+  patient,
+  resources,
+  dateFrom,
+  dateTo,
+  conversionType,
+}: {
+  bundle: Bundle<Resource>;
+  patient: Pick<Patient, "id" | "cxId">;
+  resources?: ResourceTypeForConsolidation[];
+  dateFrom?: string;
+  dateTo?: string;
+  conversionType: ConsolidationConversionType;
+}): Promise<string> {
+  const lambdaName = Config.getFHIRToMedicalRecordLambdaName();
+
+  const payload = {
+    bundle,
+    patientId: patient.id,
+    cxId: patient.cxId,
+    resources,
+    dateFrom,
+    dateTo,
+    conversionType,
+  };
+
+  const lambdaResult = await lambdaClient
+    .invoke({
+      FunctionName: lambdaName,
+      InvocationType: "RequestResponse",
+      Payload: JSON.stringify(payload),
+    })
+    .promise();
```

Comment:
Can we apply factory here and have a local implementation? We should strive to do this as much as possible, this way we can code local without relying on the external service.

Ideally, the lambda would use the same/most of the code of the local implementation, shared through the `core` package.

See these for an example:
- `makeSearchServiceIngest()` returns a "direct" implementation for local or a "sqs" one for when running on cloud envs
- the api uses
    - the "direct" implementation when local
    - the "sqs" one when running on cloud env
- the lambda `sqs-to-opensearch-xml.ts` uses the "direct" implementation (`openSearch.ingest()`)

With that, we have only one code to ingest content on OpenSearch, used by both the lambda and the api when local. Not only easier to test/work local, but also the code we run on the lambda is the same we tested local.

Reply:
Im going to take note of this and work on it right after i deploy this change just we can get this PR out to circle today

Reply:
This is https://github.com/metriport/metriport-internal/issues/1064, right? I'm working on that one.